app/controllers/admin/targets_controller.rb

```diff
@@ -39,9 +37,9 @@ def destroy
         redirect_to(admin_conference_targets_path(conference_id: @conference.short_title),
                     notice: 'Target successfully destroyed.')
       else
-        redirect_to(admin_conference_targets_path(conference_id: @conference.short_title),
-                    error: 'Target was successfully destroyed.' \
-                    "#{@target.errors.full_messages.join('. ')}.")
+        redirect_to admin_conference_targets_path(conference_id: @conference.short_title),
+                    flash:  { error: "Could not delete target for #{@conference.title}: "\
+                    "#{@target.errors.full_messages.join('. ')}." }
```

Comment:
Yes, you are right, I thought we had implemented that already. Why don't we add error flash type instead? So that our redirect is cleaner and looks like the ones with alert or notice?
http://api.rubyonrails.org/classes/ActionController/Flash/ClassMethods.html#method-i-add_flash_types